lfucg-exactions/client/js/components/AccountLedgerSummary.js

```diff
@@ -130,7 +130,7 @@ class AccountLedgerSummary extends React.Component {
                                             </div>
                                             <div className="col-xs-12">
                                                 <p className="col-xs-12">Lot Address: {accountLedgers.lot.address_full}</p>
-                                                <p className="col-md-4 col-xs-6">Current Exactions: {accountLedgers.lot && accountLedgers.lot.lot_exactions && accountLedgers.lot.lot_exactions.current_exactions}</p>
+                                                <p className="col-md-4 col-xs-6">Current Exactions: {accountLedgers.lot.lot_exactions && accountLedgers.lot.lot_exactions.current_exactions}</p>
```

Comment:
why no check for accountLedgers.lot.lot_exactions?

Reply:
It is checking for accountLedgers.lot above